### ISSUE TITLE
Operator doesn't update vmagent config if additionalScrapeConfigs secret was changed

### ISSUE THREAD

**Issue by symmetriac** [VictoriaMetrics/operator#462]
Operator (since v0.22.0) doesn't update vmagent config when secret with additional scrape config was changed.
Config describe as

```
additionalScrapeConfigs:
    name: additional-scrape-config
    key: additional-config.yaml
```

Operator v0.21.0 writes log messages when secret was updated
```
{"level":"info","ts":1649238529.6811318,"logger":"controllers.VMAgent","msg":"Reconciling","vmagent":"monitoring/vmagent-ha"}
{"level":"info","ts":1649238529.9882824,"logger":"factory","msg":"current VMAgent configuration has changed"}
{"level":"info","ts":1649238529.988304,"logger":"factory","msg":"updating VMAgent configuration secret"}
```
but operator v0.22.0 and higher doesn't

**Comment by symmetriac** [comment]
additional scrape config secret metadata:
```
finalizers:
  - apps.victoriametrics.com/finalizer
  name: additional-scrape-config
  namespace: monitoring
  ownerReferences:
  - apiVersion: operator.victoriametrics.com/v1beta1
    blockOwnerDeletion: false
    controller: true
    kind: VMAgent
    name: vmagent-ha
    uid: 938da2ae-a9a6-4ba3-85c6-ece7e5d7b380
```

**Comment by f41gh7** [comment]
I think, it should be fixed at the next release. Operator will resync configuration with configured interval. It's more reliable then watching for ownership.  Related PR https://github.com/VictoriaMetrics/operator/commit/18031e310c8f3e582257ccbbba6890f59c1d32b6

**Comment by f41gh7** [comment]
Changes was included to the v0.24.0 release. Feel free to re-open the issue if it doesn't help.

**Comment by symmetriac** [comment]
It did work, thanks
